### PR TITLE
some minor refine

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve?

some minor refine

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: hack/update_install_script.sh [hunk 1]
```diff
@@ -59,4 +59,3 @@ chmod +x $install_scirpt
 
 rm -rf $tmp_file
 rm -rf $tmp_file.bak
-rm -rf $tmp_install_scipt
```
Comment (WangXiangUSTC) [inline hack/update_install_script.sh:62]: already mv in L57

File: pkg/bpm/bpm.go [hunk 2]
```diff
@@ -230,24 +233,21 @@ type ProcessBuilder struct {
...
+	return fmt.Sprintf("%s/%d/ns/%s", DefaultProcPrefix, pid, string(typ))
 }
 
-// SetPidNS sets the pid namespace of the process
-func (b *ProcessBuilder) SetPidNS(nsPath string) *ProcessBuilder {
-	return b.SetNS([]nsOption{{
-		Typ:  PidNS,
-		Path: nsPath,
+// SetNS sets the namespace of the process
+func (b *ProcessBuilder) SetNS(pid uint32, typ NsType) *ProcessBuilder {
```
Comment (WangXiangUSTC) [inline pkg/bpm/bpm.go:242]: we use `GetNsPath` to get the path, and then use the path as an argument in `SetNetNS` and `SetPidNS`, and we can use `SetNS` directly.

**Review by cwen0 — APPROVED** [review]
LGTM

**Review by fewdan — APPROVED** [review]
LGTM

**Pull request merged**